pkgs/applications/office/sozi/default.nix

```diff
@@ -0,0 +1,38 @@
+{ appimageTools, lib, fetchurl }:
+let
+  pname = "sozi";
+  version = "23.7.25-1690311612";
+
+  src = fetchurl {
+    url = "https://github.com/sozi-projects/Sozi/releases/download/v23.07/Sozi-23.7.25-1690311612.AppImage";
+    sha256 = "sha256-QHvgevv60ZTkkdt+IWaCuXt0XVnhe5Q5oROwa2LFie8=";
+  };
+
+  appimageContents = appimageTools.extract {
+    name = "${pname}-${version}";
+    inherit src;
+  };
+in appimageTools.wrapType2 {
+  inherit pname version src;
+
+  extraPkgs = pkgs: with pkgs; [ polkit udev ];
+
+  extraInstallCommands = ''
+    mv $out/bin/${pname}-${version} $out/bin/${pname}
```

Comment:
Please note that there is an issue that tracks [superfluous use of `pname`](https://github.com/NixOS/nixpkgs/issues/277994), please double check the issue and this PR and possibly avoid uses of `pname` in the `extraInstallCommands` phase.

Reply:
I dont understand

should I just hardcode

```
    mv $out/bin/sozi-${version} $out/bin/sozi

    # Install desktop integration
    install -m 444 -D ${appimageContents}/sozi.desktop $out/share/applications/sozi.desktop
    substituteInPlace $out/share/applications/sozi.desktop \
      --replace 'Exec=AppRun' 'Exec=sozi'
    cp -r ${appimageContents}/usr/share/icons $out/share
```

?

Reply:
Yes, please 🙂